modules/test-ui/src/utils/identifiers.ts

```diff
@@ -0,0 +1,10 @@
+import { PublicIdentifier } from "@connext/vector-types";
+import bs58check from "bs58check";
+import { bufferToHex, decompress } from "eccrypto-js";
+
+export const INDRA_PUB_ID_PREFIX = "indra";
+
+export const getPublicKeyFromPublicIdentifier = (publicIdentifier: PublicIdentifier) =>
```

Comment:
This function should already be in `@connext/vector-utils`, just go ahead and import it from there.